.github/CODEOWNERS

```diff
@@ -24,6 +24,7 @@
 
 # Management Plane
 /**/resource-manager/        				@yaohaizh
+/sdk/**/azure-mgmt-*/               @yaohaizh
```

Comment:
nit: can you align the spacing with the rest of the file?  It looks like you're off by a tab or two

Reply:
Sure. It's actually a few tabs on line above. I will change to spaces for all.